src/errors.rs

```diff
@@ -33,28 +33,14 @@ pub enum InternalError {
     InternalInvariantFailed,
     #[error("Paillier error: `{0}`")]
     PaillierError(#[from] paillier::Error),
-    #[error("Failed to convert BigNumber to k256::Scalar, as BigNumber was not in [0,p)")]
-    CouldNotConvertToScalar,
-    #[error("Could not invert a Scalar")]
-    CouldNotInvertScalar,
     #[error("Reached the maximum allowed number of retries")]
     RetryFailed,
     #[error("This Participant was given a message intended for somebody else")]
     WrongMessageRecipient,
```

Comment:
User error or ProtocolError? Maybe it depends on how a Message is constructed from bytes over the wire?

Reply:
Oops,I should have deleted this when I made the separate `CallerError` in #235. This shouldn't be used anywhere any more, but it's a caller error now because it's the responsibility of the calling application to route messages to the right participant.